enterprise/models.py

```diff
@@ -4484,7 +4484,7 @@ class EnterpriseGroup(TimeStampedModel, SoftDeletableModel):
     """
     uuid = models.UUIDField(primary_key=True, default=uuid4, editable=False)
     name = models.CharField(
-        max_length=25,
+        max_length=250,
```

Comment:
Isn't the default for a charfield 255? Is there a reason it's 250?

Reply:
updated!